noir_stdlib/src/eddsa.nr

```diff
@@ -11,6 +13,28 @@ pub fn eddsa_poseidon_verify(
     signature_r8_y: Field,
     message: Field
 ) -> bool {
+    let mut hasher = PoseidonHasher::default();
+    eddsa_verify_with_hasher(
+        pub_key_x,
+        pub_key_y,
+        signature_s,
+        signature_r8_x,
+        signature_r8_y,
+        message,
+        &mut hasher
+    )
+}
+
+pub fn eddsa_verify_with_hasher<H>(
```

Comment:
```suggestion
pub fn eddsa_verify<H>(
```

Do we need to specify this?

Reply:
I think it is related to the previous point. If we need to provide the hasher, the name _with_hasher makes more sense to me, but I agree we do not need that if we can resolve the previous point.

Reply:
I resolve the conversation because I created an issue for it: #4514